### PR TITLE
Filterline: Add 'show help' button

### PR DESCRIPTION
**Pull request by larsjohnsen**
Opens the `filterlineVisible` featureTip guider

Tested in browser: Chrome 58

### CODE REVIEW

File: lib/modules/filteReddit/Filterline.js [hunk 1]
```diff
@@ -62,6 +63,9 @@ export default class Filterline {
 								<span>Hide currently filtered</span>
 							</label>
 						</div>
+						<div class="res-filterline-show-help">
+							Usage information
```
Comment (andytuba) [inline lib/modules/filteReddit/Filterline.js]: i18n

**Comment by andytuba** [comment]
Great idea, hopefully forestalls a lot of the r/enhancement posts!

File: lib/css/modules/_filteReddit.scss [hunk 2]
```diff
@@ -199,9 +200,17 @@ body.hideOver18 {
...
+	&-export-customfilters,
+	&-show-help {
 		text-align: center;
-		padding: 2px 0;
+		padding: 4px;
 		cursor: pointer;
 	}
+
+	&-show-help {
+		&::before  {
```
Comment (mc10) [inline lib/css/modules/_filteReddit.scss]: Extra space here.

File: lib/modules/filteReddit/Filterline.js (hunk 1 above)
Comment (larsjohnsen) [inline lib/modules/filteReddit/Filterline.js]: Probably superfluous at this stage, as the help tip nor any of the Filterline stuff is translated yet.

**Pull request merged**